docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,494 @@
+Please read the [PEtab.jl documentation](https://sebapersson.github.io/PEtab.jl/dev/Avaible_optimisers/#options_optimisers) to learn how to customise the various optimisers' properties. 
+
+### [Optimisation path recording](@id petab_optimisation_path_recording)
+
+To record all the parameter sets evaluated (and their objective values) during the optimisation procedure, provide the `save_trace=true` argument to `calibrate_model` (or `calibrate_model_multistart`). This is required for the various [optimisation evaluation plots](@ref petab_plotting) provided by PEtab.jl. If desired, this information can be accessed in the calibration output's `.xtrace` and `.ftrace` fields.
+
+
+## Objective function extraction
+While PEtab.jl provides various tools for analysing the objective function generated by `PEtabODEProblem`, it is also possible to extract this function for customised analysis. Given a `PEtabODEProblem`
+```@example petab1
+petab_problem = PEtabODEProblem(petab_model)
+nothing # hide
+```
+We can find the:
+1. Objective function in the `petab_problem.compute_cost`. It takes a single argument (`p`) and returns the objective value.
+2. Gradient in the `petab_problem.compute_gradient!` field. It takes two arguments (`g` and `p`) with the updated gradient values being written to `g`.
+3. Hessian in the `petab_problem.compute_hessian` field. It takes two arguments (`H` and `p`) with the updated hessian values being written to `H`.
+
+
+## [Multi-start optimisation](@id petab_multistart_optimisation)
+To avoid the optimisation process returning a local minimum, it is often advised to run it multiple times, using different initial guesses. PEtab.jl supports this through the `calibrate_model_multistart` function. This is identical to the `calibrate_model` function, but takes two additional arguments:
+
+1. `n_multistart`: The number of runs to perform.
+2. `dir_save`: A location to which the output is saved. If `dir_save=nothing`, no output is saved.
+
+And one additional optional argument:
+
+3. `sampling_method`: Selects the sampling method with which to select the initial guesses (`QuasiMonteCarlo.LatinHypercubeSample()` used by default). 
+ 
+Because `calibrate_model_multistart` handles initial guess sampling, unlike for `calibrate_model`, no initial guess has to be provided. 
+
+Here, we fit parameters through 10 independent optimisation runs, using QuasiMonteCarlo's `SobolSample` method, and save the result to the OptimizationRuns folder:
+```@example petab1
+using Optim
+using QuasiMonteCarlo
+res_ms = calibrate_model_multistart(petab_problem, IPNewton(), 10, "OptimizationRuns"; sampling_method=QuasiMonteCarlo.SobolSample())
+```
+The best result across all runs can still be retrieved using `get_ps(res_ms, petab_problem)`, with the results of the individual runs being stored in the `res_ms.runs` field. 
+
+To load the result in a later session, we can call:
+```@example petab1
+res_ms = PEtabMultistartOptimisationResult("OptimizationRuns")
```

Comment:
Does this require saving the data? If so, you should probably show how to do that too. Otherwise how does it find the results on disk?

Reply:
This was briefly described in the previous section, however, I will add a link to that.

Reply:
They are automatically saved using the `dir_save` option. It is described in the section, but I can elaborate a bit here.

Reply:
Maybe just a reminder since I missed that.

Reply:
I added extra stuff to clarify this.